GitHubUserSearch/GitHubUserSearch/Model/Model.swift

```diff
@@ -40,19 +43,19 @@ class Model {
    var currentUser:UserData?
    lazy var repositories:[RepositoryData] = [RepositoryData]()
    
-   //MARK: - memory
+   // MARK: - memory
    
    func didReceiveMemoryWarning() {
       cache.removeAllObjects()
       cleanUserSearchResults()
    }
    
-   //MARK: -
+   // MARK: -
    func setUserSearchResultsPerPage(_ count:Int) {
       apiCaller.usersPerPage = min(count, 40)
    }
-   //MARK: -
-   func searchForUser(name:String, completion:(()->())? ) {
```

Comment:
You can make this method shorter
`func search(for name: String, completion: (()->())? )` + you can make type alias for `() -> ()`

Reply:
yes, but the method @search ( for )@  will be less readable outside  - You will have to dive deeper into the context.